models/experimental/functional_mistral/tt/ttnn_functional_transformer_block.py

```diff
@@ -0,0 +1,49 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import ttnn
+import torch
+import tt_lib
+from models.experimental.functional_mistral.tt.ttnn_functional_rms_norm import rms_norm
+from models.experimental.functional_mistral.tt.ttnn_functional_attention import attention
+from models.experimental.functional_mistral.tt.ttnn_functional_feed_forward import feed_forward
+
+
+def transformer_block(
+    config,
+    x,
+    bcast_freq_xq: tt_lib.tensor.complex_tensor,
+    bcast_freq_xk: tt_lib.tensor.complex_tensor,
+    positions,
+    mask,
+    seqlen,
+    parameter,
+    device,
+    memory_config,
+):
+    att_weight = ttnn.to_layout(
+        ttnn.to_device(ttnn.from_torch(parameter.attention_norm.weight, dtype=ttnn.bfloat16), device),
+        layout=ttnn.TILE_LAYOUT,
+    )
```

Comment:
Please do this instead:

```
att_weight = ttnn.from_torch(parameter.attention_norm.weight, dtype=ttnn.bfloat16, layout=ttnn.TILE_LAYOUT, device=device)
```

Also, why is it even necessary. This is a parameter which means we should preprocess it ahead of time. What `torch.nn.module` is this? RmsNorm?x

I might need to update the code in model_preprocessing to convert torch tensors to ttnn tensors. In the meantime, can you use `custom_preprocessor` to convert these weight to `ttnn` tensor in tile layout ahead of time?